new-dti-website-redesign/src/app/sponsor/SponsorTable.tsx

```diff
@@ -0,0 +1,213 @@
+
+import { useState } from 'react';
+import Image from 'next/image';
+import benefitData from './benefits.json';
+import useScreenSize from '../../hooks/useScreenSize';
+import IconWrapper from '../../components/IconWrapper';
+import CheckIcon from '../../components/icons/CheckIcon';
+
+type Tier = 'bronze' | 'silver' | 'gold' | 'platinum';
+
+const medals = {
+  bronze: {
+    name: 'bronze',
+    link: '/sponsor/medals/bronze.svg',
+    title: 'Bronze',
+    color: '#AC773A'
+  },
+  silver: {
+    name: 'silver',
+    link: '/sponsor/medals/silver.svg',
+    title: 'Silver',
+    color: '#B2B2B2'
+  },
+  gold: {
+    name: 'gold',
+    link: '/sponsor/medals/gold.svg',
+    title: 'Gold',
+    color: '#DBA10B'
+  },
+  platinum: {
+    name: 'platinum',
+    link: '/sponsor/medals/platinum.svg',
+    title: 'Platinum',
+    color: '#7D7D7D'
+  }
+};
+const { benefits } = benefitData;
+
+const medalHeight = 57;
+const medalWidth = 45;
+const mostPopular = 'gold';
+
+const tiers = Object.keys(medals);
+
+const SponsorshipTableMobile = () => {
+  const [selectedMedal, setSelectedMedal] = useState<Tier>('bronze');
+
+  return (
+    <>
+      <div className="py-8 px-4">
+        <h2>
+          Sponsorship <br />
+          benefits
+        </h2>
+      </div>
+      <table className="w-full border-t border-border-1">
+        <thead>
+          <tr>
+            {Object.keys(medals).map((medal, index) => (
+              <th
+                key={medal}
+                className={`relative w-1/4 border-border-1 text-center ${
+                  index !== 0 ? 'border-l' : ''
+                }`}
+              >
+                <button
+                  onClick={() => setSelectedMedal(medal as Tier)}
+                  aria-label={`Show ${medal} tier`}
+                  className={`w-full py-4 transition-colors duration-200 outline-none
+                    hover:bg-background-2
+                    focus-visible:ring-2 focus-visible:ring-white
+                    ${selectedMedal === medal ? 'bg-background-2 relative' : ''}
+                  `}
+                >
+                  {selectedMedal === medal && (
+                    <div className="absolute border-b-2 bottom-0 w-full h-[1.9px] shadow-[0_-4px_8px_0_#fff]" />
+                  )}
+                  <div className="flex flex-col gap-2 items-center">
+                    <Image
+                      src={medals[medal as Tier].link}
+                      alt={medal}
+                      width={medalWidth}
+                      height={medalHeight}
+                    />
+                    <p style={{ color: medals[medal as Tier].color }}>
+                      {medals[medal as Tier].title}
+                    </p>
+                  </div>
+                </button>
+                {medals[medal as Tier].name === mostPopular && (
+                  <div className="absolute text-black -top-[38.5px] w-full bg-white rounded-tl-[8px] rounded-tr-[8px] py-[6px]">
```

Comment:
```suggestion
                  <div className="absolute text-black -top-[38.5px] w-full bg-white rounded-tl-lg rounded-tr-lg py-[6px]">
```

We can use the Tailwind `lg` utility for the border radius so that it's more consistent. And that way if ever end up updating the border radius along the whole website, we can just do so with `lg` instead of manually in every file

Reply:
Also to quickly check what Tailwind utility (`sm`, `md`, `lg`, etc.) corresponds to the value, you can hover over the class and VSC will show you the equivalent in pixels:

<img width="639" height="216" alt="Image" src="https://github.com/user-attachments/assets/5df82b6b-0e5b-4fcf-8d99-75e854847c7b" />